### ISSUE TITLE
[Question] High heat of side thrusters

### ISSUE THREAD

**Issue by Sullos** [tsunamayo/Starship-EVO#2323]
### Short Description of the issue:

Recently a few people in the community and I have noticed that our ship's heat has skyrocketed (pun intended). Ships that were finished and balanced suddenly had heat much closer to 100% or in some cases went over 100%. Have you recently changed how much heat side thrusters give off? As they now produce a lot more heat than they used to. Or is this possibly a bug.

Thanks Tsuna

**Comment by tsunamayo** [comment]
This must be a bug, you confirm it comes from side thruster? Also since what update?
Thanks

**Comment by Sullos** [comment]
I've tried confirming with the community when this change appeared, all I remember is that it happened within the last week, but more than a few days ago.
I am pretty sure its the side thruster, as I hooked switches up to all systems on the ship, and disabled them bit by bit. Once I disabled side thrusters, the ship's heat dropped a very large amount. So upon looking at them, a 3x3x2 side thruster produced heat at a similar level to a 16x8x12 large thruster. Sorry for the rough numbers, they are just roughly the dimensions of the thrusters I looked at, at the time.
I don't know if those heat figures are meant to be that way or not. But it makes it difficult to justify using side thrusters on a fighter when a large thruster block is much more heat efficient.

**Comment by Starwhip** [comment]
![20200605224037_1](https://user-images.githubusercontent.com/36343006/83978720-68541f80-a8d7-11ea-8f6a-8a6a66e6623c.jpg)
![20200605224118_1](https://user-images.githubusercontent.com/36343006/83978721-69854c80-a8d7-11ea-98f4-c9d26c3f24b9.jpg)

Noticed this issue as well, the ~3x3x3 small thrusters I had originally placed as decoration use up around 40% of the ship's heat capacity, significantly more than a bigger large block engine.

**Comment by tsunamayo** [comment]
Okay there must be something fishy because I dont reproduce that at all. My value is 169h for a 3x3x3 small thrusters, and they produce 25% heat of a large one.
@Sullos  @Starwhip could you send me a blueprint so I can investigate? Thanks

**Comment by tsunamayo** [comment]
@Sullos hey mate could please tell me if you still see this issue? I would like to fix it before ea as it is quite a serious one. Thanks a lot!

**Comment by Sullos** [comment]
Hey Tsuna, so the issue is gone!
I checked the same ship that I first saw the issue on, and it was at 79% (all the time) and that was after I went and cut out a large number of my side thrusters too. 
I looked at it again just now, and the heat of my whole ships is now at 39%. So all seems fixed!

One thing to note - I used to be able to look at thrusters and see a technical readout, or at least I think I could. It would tell me the number of thruster units per side thruster. So if I dragged it 1m^2, it would say 64 units. I could also see stats like the heat output of the thruster, but not anymore. Did you disable this overlay or is there another way of viewing it? 
Perhaps by using a link tool and clicking on a thruster the overlay with technical readouts can appear.
Testing all other things you asked me to test now, so hang in there whilst I test everything else. But this issue seems fixed!
Thanks
